config/config.go

```diff
@@ -92,7 +92,8 @@ type Project struct {
 	FPM     FPM      `yaml:",omitempty"`
 
 	// test only property indicating the path to the dist folder
-	Dist string `yaml:"-"`
+	Dist                 string `yaml:"-"`
+	SnapshotNameTemplate string `yaml:"snapshot_name_template"`
```

Comment:
maybe put it in `snapshot.name_template`, if we need to add more config for snapshots later, it would be easier...